configuration/configuration.go

```diff
@@ -57,6 +57,9 @@ type Configuration struct {
 			// contain the private portion for the file specified in
 			// Certificate.
 			Key string `yaml:"key,omitempty"`
+
+			// Specifies the CA cert for client authentication
+			CACert string `yaml:"ca,omitempty"`
```

Comment:
probably not. Other services (Apache, nginx) tend to only accept one. 

I'll do the key rename though